helpdesk-bot/pom.xml

```diff
@@ -103,11 +123,39 @@
             <version>1.58</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.bouncycastle</groupId>
```

Comment:
The same dependency declared twice conflicting versions.

Reply:
Done.